pandas/core/generic.py

```diff
@@ -4275,7 +4275,29 @@ def get_ftype_counts(self):
 
     @property
     def dtypes(self):
-        """Return the dtypes in this object."""
+        """
+        Return the dtypes in this object.
+
+        Notes
+        -----
+        It returns a Series with the data type of each column.
```

Comment:
Create a 'Returns' section to explain the output. Notes should be used only to explain technical details about the implementation of the algorithm or function behavior.